bigquery/google/cloud/bigquery/client.py

```diff
@@ -1296,7 +1304,10 @@ def load_table_from_dataframe(
         os.close(tmpfd)
 
         try:
-            dataframe.to_parquet(tmppath)
+            if job_config.schema:
```

Comment:
Note from chat: if schema isn't populated, we might want to call get_table and use the table's schema if it the table already exists and we're appending to it. (This is what pandas-gbq does)

Reply:
Ditto. Filed https://github.com/googleapis/google-cloud-python/issues/8142. I think this would make a good feature, but shouldn't block this PR.